swarm/network/stream/common_test.go

```diff
@@ -69,19 +69,13 @@ func init() {
 	log.Root().SetHandler(log.LvlFilterHandler(log.Lvl(*loglevel), log.StreamHandler(colorable.NewColorableStderr(), log.TerminalFormat(true))))
 }
 
-func createGlobalStore() (string, *mockdb.GlobalStore, error) {
-	var globalStore *mockdb.GlobalStore
-	globalStoreDir, err := ioutil.TempDir("", "global.store")
-	if err != nil {
-		log.Error("Error initiating global store temp directory!", "err", err)
-		return "", nil, err
-	}
-	globalStore, err = mockdb.NewGlobalStore(globalStoreDir)
+func createGlobalStore() (*mockdb.GlobalStore, error) {
+	globalStore, err := mockdb.NewGlobalStore()
```

Comment:
As I wrote in the comment bellow, this is invalid use of `mock/db`, there is `mock/mem` that provides in-memory implementation.